### ISSUE TITLE
ARM64 now has x64 emulation

### ISSUE THREAD

**Issue by TheCakeIsNaOH** [chocolatey/choco#2172]
Microsoft has released x64 emulation for ARM64 in preview builds.
https://blogs.windows.com/windows-insider/2020/12/10/introducing-x64-emulation-in-preview-for-windows-10-on-arm-pcs-to-the-windows-insider-program/

Thus, it is no longer always correct for `Get-OSArchitectureWidth` to always return 32 bit on an ARM64 system.
https://github.com/chocolatey/choco/blob/master/src/chocolatey.resources/helpers/functions/Get-OSArchitectureWidth.ps1#L34-L37

**Comment by riverar** [comment]
I had questions about how to deploy natively on arm64 and was pointed to the repository. Found this issue. Am a little surprised at the issue age.

**Comment by TheCakeIsNaOH** [comment]
@riverar Currently, the Chocolatey CLI PowerShell helpers do not have any built in support for native ARM/ARM64 software. This issue is about adding support for amd64 running emulated on arm64, now that emulation has been added. You want issue #1803 instead.

**Comment by riverar** [comment]
Both really. If I don't choose to package arm64 binaries, I think Chocolatey should at least use the x64 executables not the x86 ones.

**Comment by cclauss** [comment]
Status?

**Comment by vexx32** [comment]
I don't believe this has had a concrete timeline set for it as yet. I know the broad scope of functionality in this area is on our radar, though. Tagging the related issue as I think this falls under that, for the moment, at least.

Lots of the issues around this are quite old and need a second look to be fleshed out better anyways, thanks for drawing attention to it! 💜

**Comment by dennisameling** [comment]
This should be a step in the right direction: https://github.com/chocolatey/choco/pull/3907

I'll test more thoroughly in the next few days, but here's a successful CI run on an arm64 runner: https://github.com/dennisameling/choco/actions/runs/26851606761/job/79187310147
